src/styles/gallery.scss

```diff
@@ -0,0 +1,71 @@
+.gallery {
+
+  &__title {
+    margin-bottom: 40px;
+  }
+
+  &__set {
+    display: grid;
+    column-gap: 20px;
+    overflow: hidden;
+
+    &--top {
+      grid-template-columns: 2fr 1fr;
+
+      @media (max-width: $breakPoint) {
+        grid-template-columns: 1fr 1fr;
+      }
+    }
+
+    &--middle {
+      grid-template-columns: 1fr 1fr;
+    }
+
+    &--bottom {
+      grid-template-columns: 1fr 2fr;
+
+      @media (max-width: $breakPoint - 1px) {
+        grid-template-columns: 1fr 1fr;
```

Comment:
why do you use $breakPoint in some places and substract 1px in others?

Reply:
Oh, I skipped this place.
Thank you!